src/com/lftechnology/java/training/niraj/fibonacci/EvenFibonacci.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * 
+ */
+package com.lftechnology.java.training.niraj.fibonacci;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * find the sum of the even-valued terms in a fibonaci series under certain
+ * limit
+ * 
+ * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+ */
+public class EvenFibonacci {
+
+	private static final Logger LOGGER = Logger.getLogger(EvenFibonacci.class
+			.getName());
+
+	public static void main(String[] args) {
+		Fibonacci fibonacciHelper = new Fibonacci();
+		int limit = 10;
+		int num1 = 1;
+		int num2 = 2;
```

Comment:
@sanishmaharjan, The scope of these variables should be just within the function it is defined on. So it might not be good to use constant for these values.